src/components/Team/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import Select from 'react-select';
+
+import TeamCard from './TeamCard';
+
+interface Person {
+  name: string;
+  class: number;
+  roles: string[];
+  image?: string;
+}
+
+interface TeamProps {
+  data: Person[];
+}
+
+const roles = ['All Roles', 'Designer', 'Developer', 'Director', 'External', 'Projects'];
+const roleOptions = roles.map(item => ({ value: item, label: item }));
+
+const currYear = new Date().getFullYear();
+const yearOptions = ['All Years', currYear + 1, currYear + 2, currYear + 3, currYear + 4, 'Alumni'].map(item => ({ value: item, label: item }));
+
+const selectStyles = {
+  control: (provided, _) => ({
+    ...provided,
+    borderRadius: '0',
+    padding: '8px 4px 8px 8px',
+  }),
+  menu: (provided, _) => ({
+    ...provided,
+    borderRadius: '0',
+  }),
+  dropdownIndicator: (provided, _) => ({
+    ...provided,
+    color: 'black',
+  }),
+  indicatorSeparator: (provided, _) => ({
+    ...provided,
+    display: 'none',
+  }),
+};
+
+function Team(props: TeamProps): JSX.Element {
+  const [year, setYear] = useState({ value: 'All Years', label: 'All Years' });
+  const [role, setRole] = useState({ value: 'All Roles', label: 'All Roles' });
+
+  // if there's no image, use 'winter.svg'
+  props.data.forEach((data) => (data.image && data.image.indexOf('/assets/') == -1 ? (data.image = `/assets/${data.image}`) : data.image) || (data.image = '/assets/winter.svg'));
```

Comment:
Can we break this line into two? So that no line is longer than 100ish characters